draft-packages/popover/KaizenDraft/Popover/styles.scss

```diff
@@ -80,6 +80,16 @@ $large: 450px;
   @include arrow($kz-color-cluny-100, $kz-color-cluny-200);
 }
 
+.guidanceBox {
+  @extend %box;
+  background: $kz-color-cluny-100;
+  border-color: $kz-color-cluny-200;
+}
+
+.guidanceArrow {
+  @include arrow($kz-color-cluny-100, $kz-color-cluny-200);
+}
+
```

Comment:
If these are identical to the informativeBox/Arrow, could we just use `.informativeBox, .guidanceBox` instead so they get updated at the same time (or at least it's more obvious two places should be affected by those changes?)